integration/kubernetes/helm-chart/alluxio/templates/config/alluxio-conf.yaml

```diff
@@ -155,6 +155,15 @@
 {{- if .Values.fuse.jvmOptions }}
   {{- $fuseJavaOpts = concat $fuseJavaOpts .Values.fuse.jvmOptions }}
 {{- end }}
+
+{{- /* ===================================== */}}
+{{- /*       ALLUXIO_PROXY_JAVA_OPTS        */}}
+{{- /* ===================================== */}}
+{{- $proxyJavaOpts := list }}
+{{- $proxyJavaOpts = print "-Dalluxio.master.hostname=${ALLUXIO_MASTER_HOSTNAME}" | append $proxyJavaOpts }}
+{{- $proxyJavaOpts = print "-Dalluxio.worker.hostname=${ALLUXIO_WORKER_HOSTNAME}" | append $proxyJavaOpts }}
+#      ALLUXIO_PROXY_JAVA_OPTS: "-Dalluxio.worker.memory.size=1G  -Dalluxio.master.hostname=alluxio-master  -Dalluxio.worker.hostname=alluxio-worker"
```

Comment:
Stale comments？

Reply:
yes, sorry missed that one. It's a copy of what I was using on a docker-compose for testing proxy a while back.